app/assets/scripts/views/imagery-search.js

```diff
@@ -93,9 +122,15 @@ var ImagerySearch = React.createClass({
 
     this.maps = [
       this.mainMap,
-      ...getImagerySearchLayers().map((o, i) => this.setupMap(this.refs[`mapThumb${i}`], o))
+      ...getImagerySearchLayers().map(
+        (o, i) => this.setupMap(this.refs[`mapThumb${i}`], o, initialMapView)
+      )
     ];
 
+    this.maps.map(
+      (map) => { map.on('moveend', this.onMapChange); }
```

Comment:
Oh! Yes, you're right. That happened because I was originally solving the jitter by using `dragend` and `zoomend` which I assume aren't broadcast to all other maps by the sync plugin. But now we have a different jitter fix I'll just bind to the main map.